qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -48,7 +48,40 @@
 
 def get_standard_gate_name_mapping():
     """Return a dictionary mapping the name of standard gates and instructions to an object for
-    that name."""
+    that name.
+    
+    For Example:
+
+    For the identity gate, the dictionary contains a mapping-
+
+    .. parsed-literal::
+        'id': Instruction(name='id', num_qubits=1, num_clbits=0, params=[])
+
+    Note here that the object on the right belongs to the broad class: 
+
+    * :class:`qiskit.circuit.Instruction`
+
+    This function returns a map of all the standard gates present in the qiskit library. 
+    Instruction object for a particular gate can be captured by indexing the map with the name 
+    of the gate.
```

Comment:
I don't think this paragraph adds a ton of value. The first sentence mostly repeats the intro paragraph. The second sentence is describing how Python dictionaries work, which we don't need to explain; the code example below it already shows off that technique. I'd remove it.

Reply:
okay. I have removed this paragraph.
Thank you